redash/utils/requests_session.py

```diff
@@ -0,0 +1,12 @@
+import requests
+from redash import settings
+
+
+class ConfiguredSession(requests.Session):
+    def request(self, *args, **kwargs):
+        if not settings.REQUESTS_ALLOW_REDIRECTS:
+            kwargs.update({"allow_redirects": False})
+        return super(ConfiguredSession, self).request(*args, **kwargs)
```

Comment:
Since we're in Python 3 now you can do this:
```suggestion
        return super().request(*args, **kwargs)
```